.github/CONTRIBUTING.md

```diff
@@ -2,13 +2,13 @@
 
 Thank you for considering contributing to SAP Fiori Fundamentals. It's people like you that make this such a great tool.
 
-### 1. Where do I go from here?
+## 1. Where do I go from here?
 
-If you've noticed a bug or have a question, [search the issue tracker][] to see if
+If you've noticed a bug or have a question, [search the issue tracker](https://github.com/SAP/fundamental/issues) to see if
 someone else in the community has already created a ticket. If not, go ahead and
-[make one][]!
+[make one](https://github.com/SAP/fundamental/issues/new/choose)!
 
-### 2. Fork & create a branch
+## 2. Fork & create a branch
 
 If this is something you think you can fix, then [fork Fundamentals][] and
```

Comment:
The "fork Fundamentals" appears a bit strange.  I'm not quite sure if this is supposed to be a link.  I don't think it _needs_ to be, but the square brackets should be removed.